### PR TITLE
Create classification

### PR DESCRIPTION
**Pull request by cupatea**
Here is the list of changes that has been implemented in this branch :
1. Categories and types have been added;
2. line_items/index and line_item_form have been updated.

Now you can choose existing category ( which are appeared after running `rails db:seed`, but I'm strongly recommend you to run `rails db:reset` instead ). Each category has boolean field "income " which means that current category belongs to income if it's true or to expense if it's false.

@verkligheten 
Please check current code and clarify what additional functionality is needed

Thx

### CODE REVIEW

File: app/views/shared/_line_item_form.html.erb [hunk 1]
```diff
@@ -7,6 +7,11 @@
   </div>
 
   <div class="field">
+    <%= f.label :category %>
+    <%= f.collection_select(:category_ids,@categories, :id, :name)%>
```
Comment (verkligheten) [inline app/views/shared/_line_item_form.html.erb]: перед @categories не хватает пробела

File: db/seeds.rb [hunk 2]
```diff
@@ -23,18 +23,18 @@
...
-#                                      {name: 'Digital'},
-#                                      {name: 'Food & Dining'},
-#                                      {name: 'Helth Care'},
-#                                      {name: 'Housing'},
-#                                      {name: 'Leisure'},
-#                                      {name: 'Loans'},
-#                                      {name: 'Taxes'},
-#                                      {name: 'Transportation'},
-#                                      {name: 'Travel'}])
+Category.create([{name: 'Automobile',income: false},
```
Comment (verkligheten) [inline db/seeds.rb]: вынести имена в масив

File: app/views/line_items/index.html.erb [hunk 3]
```diff
@@ -8,30 +8,34 @@
...
-                <% delete_icon = '<i class="small material-icons">delete</i>' %>
-                <% edit_icon = '<i class="small material-icons">mode_edit</i>' %>
-                <% view_icon = '<i class="small material-icons">info_outline</i>' %>
-                <%= link_to view_icon.html_safe, line_item_path(line_item) %>
-                <%= link_to edit_icon.html_safe, edit_line_item_path(line_item) %>
-                <%= link_to delete_icon.html_safe, line_item_path(line_item), method: :delete %>
-              </td>
-            </tr>
-          <% end %>
+          <% symbol = line_item.get_currency_symbol %>
```
Comment (verkligheten) [inline app/views/line_items/index.html.erb]: избавиться от переменных во view

**Comment by cupatea** [comment]
I've fix all the mentioned remarks and also have added balance function which calculate sum of all incomes and expenses in each bank account.

@verkligheten
May we make a merge?

Thx

File: app/views/bank_accounts/index.html.erb [hunk 4]
```diff
@@ -21,13 +22,17 @@
...
+              <td><%= bank_account.balance %></td>
               <td>
-                <% delete_icon = '<i class="small material-icons">delete</i>' %>
-                <% edit_icon = '<i class="small material-icons">mode_edit</i>' %>
-                <% view_icon = '<i class="small material-icons">info_outline</i>' %>
-                <%= link_to view_icon.html_safe, bank_account_path(bank_account) %>
-                <%= link_to edit_icon.html_safe, edit_bank_account_path(bank_account) %>
-                <%= link_to delete_icon.html_safe, bank_account_path(bank_account), method: :delete %>
+                <%= link_to bank_account_path(bank_account) do %>
+                  <%= content_tag :i, "info_outline", class: "small material-icons" %>
```
Comment (verkligheten) [inline app/views/bank_accounts/index.html.erb]: content_tag - использовать внутри хелпера

File: app/views/devise/confirmations/new.html.erb [hunk 5]
```diff
@@ -9,7 +9,7 @@
   </div>
 
   <div class="actions">
-    <%= f.submit "Resend confirmation instructions" %>
+    <%= f.button "Resend confirmation instructions" %>
```
Comment (verkligheten) [inline app/views/devise/confirmations/new.html.erb]: названия кнопок вынести в переводы и использовать i18n

File: db/seeds.rb [hunk 6]
```diff
@@ -4,37 +4,21 @@
...
-#                          {name: 'Recurring'}])
+incomes = ['Salary & Wages','Investments','Other']
+expanses = ['Automobile','Bills','Digital','Food & Dining','Helth Care','Housing','Leisure','Loans','Taxes','Transportation','Travel']
 
-#line_item_type = LineItemType.create([{name: 'Income'},
-#                                      {name: 'Expense'}])
-#income = line_item_type.first
-#expense = line_item_type.second
+incomes.each do |income|
+  Category.create [{name: income, income: true}]
```
Comment (verkligheten) [inline db/seeds.rb]: 1. зачем тут массив и хеш?
2. очень хреново выбрано название income. это вообще логично "name: income, income: true"?

**Comment by cupatea** [comment]
@verkligheten 
I can't get it, why I18n is not working for index actions. The only way it works  - when you're adding /en/ before path, e.g.:
`http://localhost:3000/en/line_items`
Without `/en/` it must take path `line_items` as parameter and decide that it is locale.
I want it works without adding of`/en/` or at least that rails helpers provide this adding of`/en/` somehow.
There must be some trick with routing or something, can you please advice?

Thx

**Comment by cupatea** [comment]
@verkligheten
I've fixed completely everything according to remarks above, please check out.
Can we merge for now as to continue to build up new futures in the others suitable branches? 

Thanks

**Pull request merged**